website/content/en/v0.27.3/concepts/node-templates.md

```diff
@@ -222,7 +222,14 @@ Select all AMIs with a specified tag:
 Select AMIs by name:
 ```yaml
   amiSelector:
-    Name: my-ami
+    aws::name: my-ami
```

Comment:
Weirdly enough, there's the name tag and the actual name of the AMI that you can select on. We should probably specify that this is the name tag on L222 but keep an example in the docs with the Name tag as well as with `aws::name`

Reply:
It works if you use `create-tags` with `Name` being the key. If you create an AMI and view this screen, you can see there's a dileniation between the "Name" (which is the name tag) and the AMI Name, which is the actual "name" attribute. I admit it's incredibly confusing, but I think we should probably callout both in the docs.

![Screenshot 2023-04-21 at 4 09 41 PM](https://user-images.githubusercontent.com/26334334/233747204-3ff9b27c-7104-447e-9b37-0b0071df0c33.png)

Reply:
@jonathan-innis  okay, just so that I add the right parts back in, am I right in understanding that adding the below block in is what you mean by use of **create tags** with name?
 
```yaml
amiSelector:
    Name: my-ami
```

Reply:
Yes, that's correct. I think the examples should be:
1. Select AMIs by Name Tag (Name)
2. Select AMIs by AMI Name (aws::name)
3. Select AMIs with a specific owner (aws::owners)
4. Select AMIs explicitly with id (aws::ids)

Reply:
Okay I have updated the docs now. Can you re-review please? ta.